### PR TITLE
Added pull_request_template.md file

### PR DESCRIPTION
**Pull request by sudeepta14**
Hi! I added the pull request template file. Please let me know if you liked it or need further modifications. Thank you.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+## Before you make a Pull Request, read the important guidelines:
+
+ <ul>
+  <li> Why is the change important? </li>
+   <li> What does this fix? </li>
+   <li> Is this a bug fix or a feature? </li>
```
[the remark this answers is not in the document]
Comment (sudeepta14) [reply .github/pull_request_template.md]: Yes, it certainly does

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## How Has This Been Tested?
+
+Please let us know if you have tested your PR and if we need to reproduce the issues. Also, please let us know if we need any relevant information for running the tests.
+
+<ul>
+ <li> Test A </li>
+ <li> Test B </li>
+</ul>
+
+**Test Configuration**:
```
[the remark this answers is not in the document]
Comment (sudeepta14) [reply .github/pull_request_template.md]: cool, i will update it

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+<ul>
+ <li> Firmware version: </li>
```
[the remark this answers is not in the document]
Comment (sudeepta14) [reply .github/pull_request_template.md]: sure, I will check for more information and add it

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ <li> Hardware: </li>
+  <li> Toolchain: </li>
+  <li> SDK: </li>
+</ul>
+
+## Things to check on:
+
+<ul>
+ <li> My Pull Request code follows the coding standards and styles of the project </li>
```
[the remark this answers is not in the document]
Comment (sudeepta14) [reply .github/pull_request_template.md]: sure

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+
+## Before you make a Pull Request, read the important guidelines:
```
[the remark this answers is not in the document]
Comment (sudeepta14) [reply .github/pull_request_template.md]: I have added issue link? Can you please tell me if I need to add something to it.

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+### Issue Link :link:
```
Comment (croossin) [inline .github/pull_request_template.md]: Some are `###` and some are `##`

Comment (sudeepta14) [reply .github/pull_request_template.md]: worked on it

**Pull request merged**